Kitodo/src/main/java/org/kitodo/production/helper/SearchResultGeneration.java

```diff
@@ -60,38 +56,34 @@ public HSSFWorkbook getResult() {
         return getWorkbook();
     }
 
-    private List<ProcessDTO> getResultsWithFilter() {
-        List<ProcessDTO> processDTOS = new ArrayList<>();
-        try {
-            processDTOS = ServiceManager.getProcessService().findByQuery(getQueryForFilter(ObjectType.PROCESS),
-                ServiceManager.getProcessService().sortById(SortOrder.ASC), true);
-        } catch (DataException e) {
-            logger.error(e.getMessage(), e);
-        }
-
-        return processDTOS;
+    private List<Process> getResultsWithFilter() {
+        List<Process> processInterfaces;
+        processInterfaces = ServiceManager.getProcessService().getByQuery(getQueryForFilter(Process.class)
+                    + " ORDER BY id ASC");
+        return processInterfaces;
     }
 
     /**
      * Gets the query with filters.
      *
-     * @param objectType Type of object that should be filtered
-     * @return A BoolQueryBuilder
+     * @param objectType
+     *            Type of object that should be filtered
+     * @return A query
      */
-    public BoolQueryBuilder getQueryForFilter(ObjectType objectType) {
-        BoolQueryBuilder query = new BoolQueryBuilder();
-
-        try {
-            query = ServiceManager.getFilterService().queryBuilder(this.filter, objectType, false, false);
-        } catch (DataException e) {
-            logger.error(e.getMessage(), e);
+    public String getQueryForFilter(Class<? extends BaseBean> objectType) {
+        String query = "FROM Process AS process";
+        String operator = " WHERE";
+        if (!StringUtils.isBlank(filter)) {
+            query += operator + " process.title LIKE '%" + filter + "%'";
+            operator = " AND";
         }
-
         if (!this.showClosedProcesses) {
-            query.mustNot(ServiceManager.getProcessService().getQueryForClosedProcesses());
+            query += operator + " process.sortHelperStatus != '100000000000'";
+            operator = " AND";
         }
         if (!this.showInactiveProjects) {
-            query.mustNot(ServiceManager.getProcessService().getQueryProjectActive(false));
+            query += operator + " process.project.active = 0";
```

Comment:
So far I know is "active" a boolean type and comparison should be done by `true` or `false` instead of `1` or `0` values.

Type checking will be more important on more recent  versions of Hibernate and / or H2 database.